### PR TITLE
stats.lua: bikeshedding

### PR DESCRIPTION
**Pull request by kasper93**

### CODE REVIEW

**Comment by hooke007** [comment]
I think it should be fixed too. https://github.com/mpv-player/mpv/issues/11544

**Comment by kasper93** [comment]
> I think it should be fixed too. https://github.com/mpv-player/mpv/issues/11544

Sorry, I'm not interested in more in depth changes, I just fixed few nitpicks that I agreed on.

**Comment by hooke007** [comment]
It's not a feature request actually. When that option is enabled, any info below is not available for any one.

File: player/lua/stats.lua [hunk 1]
```diff
@@ -736,29 +741,31 @@ local function add_video(s)
...
         append_resolution(s, ro, "Output Resolution:", "dw", "dh")
     end
-    append_resolution(s, {w=scaled_width, h=scaled_height}, "Scaled Resolution:")
+    local scale = nil
     if not mp.get_property_native("fullscreen") then
-        append_property(s, "current-window-scale", {prefix="Window Scale:"})
+        scale = mp.get_property_native("current-window-scale")
     end
+    append_resolution(s, {w=scaled_width, h=scaled_height, s=scale}, "Scaled Resolution:")
+    append_property(s, "deinterlace", {prefix="Deinterlacing:"})
```
Comment (sfan5) [inline player/lua/stats.lua:750]: should be hidden if not deinterlacing IMO

File: player/lua/stats.lua [hunk 2]
```diff
@@ -736,29 +741,31 @@ local function add_video(s)
...
+
     append(s, r["pixelformat"], {prefix="Pixel Format:"})
     if r["hw-pixelformat"] ~= nil then
         append(s, r["hw-pixelformat"], {prefix_sep="[", nl="", indent=" ",
                 suffix="]"})
     end
+    append(s, r["colorlevels"], {prefix="Levels:", nl=""})
```
Comment (sfan5) [inline player/lua/stats.lua:757]: maybe move it to the same line as the other color stuff?

**Comment by sfan5** [comment]
Suggestion:
grafik
I think the "1.78:1" is pretty pointless in this case so I propose the following trick:
1. Check if aspect-name consists of only numbers and punctuation
2. If so, suppress the `%f:1` and display only aspect name

File: player/lua/stats.lua (hunk 1 above)
Comment (kasper93) [reply player/lua/stats.lua]: Fair, done.

File: player/lua/stats.lua (hunk 2 above)
Comment (kasper93) [reply player/lua/stats.lua:760]: I think it fits nicely along pixel format. And the next line is pushing its limits if it comes to length, so I would avoid adding more to it.

**Comment by kasper93** [comment]
>  I think the "1.78:1" is pretty pointless in this case so I propose the following trick:

I would like to keep it consistent format between all others. 1.78:1 is still valid representation of 16:9 aspect. If lil redundant, but it doesn't switches format unexpectedly.

**Review by sfan5 — APPROVED** [review]

**Pull request merged**